views/components/colorPicker.jsx

```diff
@@ -0,0 +1,42 @@
+import { Paper, Typography } from "@material-ui/core"
+import React, { useCallback, useRef, useState } from "react"
+import { HexColorPicker } from "react-colorful"
+
+import useClickOutside from "../hooks/useClickOutside"
+
+const ColorPicker = ({ 
```

Comment:
❓ The other components in this project have PascalCase file names, right? Should this file name be changed to match that pattern (i.e., colorPicker.jsx => ColorPicker.jsx)